target/linux/ramips/image/mt7621.mk

```diff
@@ -2524,6 +2524,22 @@ define Device/planex_vr500
 endef
 TARGET_DEVICES += planex_vr500
 
+define Device/plasmacloud_pax1800-lite
+  $(Device/dsa-migration)
```

Comment:
I thought this is only required for devices that were added before the DSA switch?